com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -466,7 +472,20 @@ public static ShaderFeatures supportedFeatures
             }
         }
 
-        private static ShaderFeatures _supportedFeatures = 0;
+        public static ShaderFeatures alwaysEnabledFeatures
```

Comment:
The name is misleading but well naming is hard. You can at least add a comment to explain this property.